### PR TITLE
Projects cards and list view with filters  [SCI-5208]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-5208](https://biosistemika.atlassian.net/browse/SCI-5208)

### What was done
Adapt filter results to cards and list view

### CODE REVIEW

File: app/assets/stylesheets/projects.scss [hunk 1]
```diff
@@ -803,6 +809,24 @@ li.module-hover {
       grid-template-columns: max-content 1fr 1fr 1fr 1fr max-content;
       margin: 40px 0 0 6px;
 
+
+      .projects-group {
+        //border-bottom: 1px solid $color-concrete;
```
Comment (okriuchykhin) [inline app/assets/stylesheets/projects.scss:814]: Is it left here intensionally?

File: app/views/projects/index/_breadcrumbs.html.erb [hunk 2]
```diff
@@ -4,7 +4,7 @@
               class: 'project-folder-link',
               data: { projects_cards_url: cards_projects_url }) %>
   <span>/</span>
-  <% @current_folder.parent_folders.each do |project_folder| %>
+  <% target_folder&.parent_folders&.each do |project_folder| %>
```
Comment (okriuchykhin) [inline app/views/projects/index/_breadcrumbs.html.erb:7]: You can write <% (@current_folder || target_folder).parent_folders.each do |project_folder| %>

Comment (urbanrotnik) [reply app/views/projects/index/_breadcrumbs.html.erb:7]: We have to keep safe navigation. When you are rendering breadcrumbs for the root folder, your target_folder/current_folder is nil

Comment (okriuchykhin) [reply app/views/projects/index/_breadcrumbs.html.erb:7]: Like this?
<% (@current_folder || target_folder)&.parent_folders&.each do |project_folder| %>

Comment (urbanrotnik) [reply app/views/projects/index/_breadcrumbs.html.erb:7]: Yes, but whenever the `@current_folder` exists, `target_folder` is there. I mean, I explicitly passing taget_fodler as a `locale` on all places where I use partial.

Comment (okriuchykhin) [reply app/views/projects/index/_breadcrumbs.html.erb:7]: <% (local_assigns[:target_folder] || @current_folder)&.parent_folders&.each do |project_folder| %>

**Review by okriuchykhin — APPROVED** [review]

**Pull request merged**